components/member-card/showskills.js

```diff
@@ -0,0 +1,36 @@
+import { skills_data } from './skills';
+import classNames from './card.module.scss';
+import PropTypes from 'prop-types';
+
+const Showskills = ({ show }) => {
+  return (
+    <div className={classNames.showskills}>
+      {skills_data.topskills.map((skill) => {
+        return (
+          <p
+            style={{ background: `${skill.background}` }}
+            className={classNames.techskills}
+            key={skill.color}>
+            {skill.tech}
+          </p>
+        );
+      })}
+      {show &&
+        skills_data.skills.map((skill) => {
+          return (
+            <p
+              style={{ background: `${skill.background}` }}
+              className={classNames.techskills}
+              key={skill.color}>
```

Comment:
why do you think using `color` as the key is a good practice? what if any two skills have the same color? can we please change to use a unique id?

Reply:
added id property.